streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -240,8 +240,8 @@
                     STATE_DIR_DOC)
             .define(REPLICATION_FACTOR_CONFIG,
                     Type.INT,
-                    1,
-                    Importance.MEDIUM,
+                    3,
```

Comment:
> REPLICATION_FACTOR_CONFIG = -1 => streams is smart and can figure it out for me.

What should this be than? Would we need another parameter "default_replacation_factor" and Streams can choose between 1 and this value? Not sure if this would not be too confusing.